pkg/cosign/verify.go

```diff
@@ -212,7 +241,7 @@ func ValidateAndUnpackCert(cert *x509.Certificate, co *CheckOpts) (signature.Ver
 			if identity.Subject != "" {
 				regex, err := regexp.Compile(identity.Subject)
```

Comment:
@vaikas - FYI, I think you added this originally

Reply:
Yeah, I sure did add it. @znewman01, in your particular example, yeah, you'd have to use:
zjn@mail\\.example\\.com

Let's take this off the thread however, and if we want this to be a non-regexp, I'd be happy to make the change. Would you mind creating an issue, tagging me in it so we have a proper track record and I'll go change it so that it is what folks want.

Thanks for the flagging!!!

Reply:
ACK will do :)